week2/2-website/css/style.css

```diff
@@ -5,6 +5,194 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+    background-color: white;
+    position: fixed;
+    top: 0;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+
+#logo {
+    display: flex;
+    justify-content: center;
+    width: 43vw;
+}
+#header-img {
+    width: 40px;
+    height: 40px;  
+}
+nav {
+    width: 57vw;
+    display: flex;
+    justify-content: flex-start;
+}
+nav ul li {
+    display: inline-block; 
+    padding: 10px 15px 10px 15px;
+    font-weight: 500;
+    
+}
+nav ul li:first-of-type {
+    font-weight: bold;
+}
+a {
+    text-decoration: none;
+    color: #888e99;
+}
+#imgcont {
+    margin-top: 0px;
+    background-image: url("/img/first-background.jpg");
+
+    background-size: cover;
+    color: ghostwhite;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    min-height: 700px;  
+    }
+h1 {
+    padding-bottom: 0;
+    color: ghostwhite;
+    font-size: 3.125rem;
+    font-weight: 300;
+    margin-bottom: 0;
+}
+p {
+    margin: 10px;
+    font-size: 1.5rem;
+    font-weight: 300;
+}
+#button {
+    text-align: center;
+    background-color: rgb(241, 87, 41);
+    font-size: 1.125rem;
+    width: 100px;
+    border-radius: 5px;
+    padding: 15px 25px;
+}
+#button a {
+    color: white;
+    font-size: 1.5;
+    font-weight: 400;
+}
+#button:hover {
+    background-color: rgb(218, 68, 23);
+}
+
+#secondSection {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+#secondSection h1 {
+    margin-top: 100px;
+    margin-bottom: 50px;
+    color: black;
+    font-size: 2.75rem;
+    font-weight: 300;
+}
+#icons {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 70%;
+    border-bottom:rgb(189, 208, 226);
+    border-width: none;
+}
+.ico {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 300px;
+    width: 30%;
+    height: auto;
+}
+.ico img {
+    width: 100%;
+    height: auto;
+    
+}
+.iconCont {
+     width: auto;
+}
+h2 {
```

Comment:
move any direct tag styling to the top of your page - for example:
```
h1, h2, h3 {
  font-weight: 500;
  color: white
}

h1 {
  font-size: 1.25rem
}

h2 {
  font-size: 1rem;
}

a {
  color: blue;
}

...etc
```

If adding something specific like `text-align` use a class or ID selector so as not to affect all `h2` tags